webots_ros2_turtlebot/launch/robot_launch.py

```diff
@@ -42,26 +43,25 @@ def get_ros2_nodes(*args):
     use_sim_time = LaunchConfiguration('use_sim_time', default=True)
 
     # TODO: Revert once the https://github.com/ros-controls/ros2_control/pull/444 PR gets into the release
```

Comment:
Just out of curiosity, this PR https://github.com/ros-controls/ros2_control/pull/444 was merged in June 2021, so we can probably address this TODO now.